src/server/controllers/account-transactions.js

```diff
@@ -261,6 +270,7 @@ const csvMapping = {
   payoutMethodType: (t) => get(t, 'expense.payoutMethod.type'),
   merchantId: (t) => get(t, 'merchantId'),
   orderMemo: (t) => get(t, 'order.memo'),
+  orderProcessedAt: (t) => formatOrderProcessedDate(t.order),
```

Comment:
```suggestion
  orderProcessedDate: (t) => formatOrderProcessedDate(t.order),
```

Reply:
We should not need an utility for this.

Reply:
Just do something like the `date` field.

Reply:
I've renamed the variable and removed the utility function.

Reply:
@znarf : Sorry, somehow missed commit due to error. I've just pushed the changes.